packages/design-system/src/components/OcCheckbox/OcCheckbox.vue

```diff
@@ -116,26 +116,20 @@ export default defineComponent({
         'oc-rounded',
         'oc-checkbox-' + getSizeClass(this.size),
         { 'oc-checkbox-outline': this.outline },
-        { 'oc-checkbox-checked': this.modelValue }
+        { 'oc-checkbox-checked': this.isChecked }
       ]
     },
     labelClasses() {
       return {
         'oc-invisible-sr': this.hideLabel,
         'oc-cursor-pointer': !this.disabled
       }
-    }
-  },
-  created() {
-    this.setChecked(this.model)
-  },
-  methods: {
-    setChecked: function (value) {
-      if (typeof value === 'boolean') {
-        this.checked = value
```

Comment:
As far as I can see, `this.checked` does not exist, so this code did not make any sense.